manager/integration/tests/test_migration.py

```diff
@@ -220,7 +223,70 @@ def test_migration_with_rebuilding_replica(clients, volume_name):  # NOQA
     7. Observe volume migrated to node 2 (single active engine).
     8. Validate initially written test data.
     """
-    return
+    # Step 1
+    client = get_random_client(clients)  # NOQA
```

Comment:
Why we need to add `# NOQA`, also this commit looks weird and not sure the purpose `sdf`?.